Resources/doc/index.rst

```diff
@@ -183,6 +183,45 @@ table of your production database. Regardless of what server you're on, you
 can always safely run this command to execute only the migrations that haven't
 been run yet on *that* particular database.
 
+Also, you can to give it a test! First lets just do a ``dry-run`` to make sure it produces the SQL we expect:
```

Comment:
`lets` is not valid.

Thus the doc on symfony.com never uses `we`, always `you`. And the doc of this bundle currently follows this rule